### ISSUE TITLE
ClientUser.setGame() doesn't work anymore.

### ISSUE THREAD

**Issue by kvn1351** [hydrabolt/discord.js#1831]
**Please describe the problem you are having in as much detail as possible:**

A bot of mine has been running without any issues for over three months now. For some reason, however, "Playing \*GAME\*" isn't showing up anymore - since yesterday(?). I've hosted the bot on multiple machines and yet the issue persists.

**Include a reproducible code sample here, if possible:**
```js
await client.user.setGame('?help')
    
--OR--
    
await client.user.setPresence({
  data: {
       game: {
            name: '?help'
        }
    }
})
```

**Further details:**

- discord.js version: 11.1.0
- node.js version: 7.10.1
- Operating system: Linux/Windows
- Priority this issue should have – please be realistic and elaborate if possible: Low

- [ ] I found this issue while running code on a __user account__
- [ ] I have also tested the issue on latest master, commit hash:

**Comment by Lewdcario** [comment]
This is a known issue as Discord now requires the 'type' parameter, even when not providing a stream. The fix would be:
client.user.setPresence({ game: { name: 'some name', type: 0 } });

**Comment by appellation** [comment]
#1811 #1807 #1804 #1782 #1777

**Comment by SpaceEEC** [comment]
Duplicate of #1811, #1807, #1804, #1798
This issue has been fixed in master and 11.1-dev already, see #1785 #1782.
If you don't want to update to the 11.1-dev branch you can use `client.user.setPresence({ game: { name: 'yourGameGoesHere', type: 0 } });` as a workaround.
